lib/services/components.js

```diff
@@ -469,8 +473,12 @@ function getStyles(name, slug) {
 function getScripts(name, slug) {
   const site = siteService.sites()[slug],
     assetDir = site && site.assetDir || mediaDir,
-    assetDirLen = assetDir.length,
-    assetPath = site && site.assetPath || '';
+    assetDirLen = assetDir.length;
+  let assetPath = site && site.assetPath || '';
+
+  if (site && site.path && site.assetPath !== site.path) {
+    assetPath = site.path;
+  }
```

Comment:
Looking at this again, I can see why it is a bit weird. It's saying:  "If there is a site that has a path, and if the path is different than the assetPath, use the path for the assetPath."
```
 if (site && site.path && site.assetPath !== site.path) {
    assetPath = site.path;
 }
```

is the same as 

```
 if (site && site.path) {
    assetPath = site.path;
 }
```

because you're saying assetPath should be the same as path if path exists.  You could combine that with the statement above to create

```
let assetPath = site && (site.path || site.assetPath || '');
```

Is that what we really want?

It seems more likely that this is a configuration error, doesn't it?